setcamera/setcamera.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "setcamera",
+	"text": "%1 enables Camera module",
```

Comment:
You can remove the workd "enables" and the text will be:
Enable Camera module (if Enable is selected) and
Disable Camera module (if Disable is selected)